CosmosApiMessageStatusChangeFeed/index.ts

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/naming-convention */ // disabled in order to use the naming convention used to flatten nested object to root ('_' char used as nested object separator)
+import * as winston from "winston";
+import { Context } from "@azure/functions";
+import { AzureContextTransport } from "@pagopa/io-functions-commons/dist/src/utils/logging";
+import { TableClient, AzureNamedKeyCredential } from "@azure/data-tables";
+import * as KP from "../utils/kafka/KafkaProducerCompact";
+import { ValidableKafkaProducerConfig } from "../utils/kafka/KafkaTypes";
+import { getConfigOrThrow } from "../utils/config";
+import { IBulkOperationResult } from "../utils/bulkOperationResult";
+import { messageStatusAvroFormatter } from "../utils/formatter/messageStatusAvroFormatter";
+import { handleMessageStatusChange } from "./handler";
+
+// eslint-disable-next-line functional/no-let
+let logger: Context["log"] | undefined;
+const contextTransport = new AzureContextTransport(() => logger, {
+  level: "debug"
+});
+winston.add(contextTransport);
+
+const config = getConfigOrThrow();
+
+const messageStatusConfig = {
+  ...config.targetKafka,
+  sasl: {
+    ...config.targetKafka.sasl,
+    password: config.MESSAGE_STATUS_TOPIC_CONNECTION_STRING
+  },
+  topic: config.MESSAGE_STATUS_TOPIC_NAME
+};
+
+const messageStatusTopic = {
+  ...messageStatusConfig,
+  messageFormatter: messageStatusAvroFormatter()
+};
+
+const kakfaClient = KP.fromConfig(
+  messageStatusConfig as ValidableKafkaProducerConfig, // cast due to wrong association between Promise<void> and t.Function ('brokers' field)
+  messageStatusTopic
+);
+
+const errorStorage = new TableClient(
+  `https://${config.ERROR_STORAGE_ACCOUNT}.table.core.windows.net`,
+  config.ERROR_STORAGE_TABLE,

```

Comment:
Using the same table for error thrown by different process, could make the error processing harder.
Maybe we could:
1 - use different table
or
2 - add a field in the error table with the process (or entity) name

As 3th options we could:
3 - leave a uniqe error table as is and use the decode to recognize the entity type from error, but this will require to write use the avro deserialization as a io-ts surrogate.

Reply:
Ok, since the table has a generic name, I thought it was meant to be generic for all exports.

Reply:
Moved to new table in [5980b15](https://github.com/pagopa/io-functions-elt/pull/35/commits/5980b15674ba1c3746998c7bc098471336e79737)